planning/route_handler/src/route_handler.cpp

```diff
@@ -911,6 +911,65 @@ lanelet::Lanelets RouteHandler::getRightOppositeLanelets(
   return lanelet_map_ptr_->laneletLayer.findUsages(lanelet.rightBound().invert());
 }
 
+lanelet::ConstLanelets RouteHandler::getAllSharedLineStringLanelets(
+  const lanelet::ConstLanelet & current_lane, bool is_right, bool is_left,
+  bool is_opposite) const noexcept
+{
+  const auto searchAllLeftLanelets = [this, &is_opposite](
```

Comment:
@zulfaqar-azmi-t4 
can you separate lambda functions to be able to use separate function?

Reply:
Yeah sure. Is the following okay?
```
func searchAllLeftLanelets
func searchAllRightLanelets

func getAllSharedLineStringLanelets(is_right, is_left){
if(is_right)
  searchAllRight
if(is_left)
  searchAllLeft
}
```

Reply:
The function is now separated.